email/1.3/modules/ROOT/pages/email-trigger.adoc

```diff
@@ -0,0 +1,219 @@
+
+
+The following example shows how to configure this parameter in the global element:
+
+. In Studio, navigate to the *Global Element* view and click *Create* to open the *Choose Global Type* window.
+. In the *Filter* box, type `email` and choose either *Email IMAP* or *Email POP3*, and click *OK*.
+. In the *Global Element Properties* window, in the main *Advanced* tab, for the *Attachment naming strategy* field select one of the following options:
+
+* *NAME*
+* *NAME_HEADERS*
+* *NAME_HEADERS_SUBJECT*
+
+. In the *General* tab, for *Connection* select any of the connection types to provide to the configuration.
+. Specify the connection information for the connector, such as *Host*, *Port* and *TLS configuration*.
+. On the subsequent *Advanced* tab, optionally specify timeout configuration and reconnection information, including a reconnection strategy.
+. Click *OK*.
+
+The following screenshot shows the *Attachment naming strategy* configuration in the global element:
+
+.Attachment naming strategy Global Element
+image::email-attachment-1.png[Attachment naming strategy Global Element]
+
+In the XML editor, the configuration `attachmentNamingStrategy="NAME"` is under the `imap-config` tag:
+
+[source, xml, linenums]
+----
+<email:imap-config name="gmail" attachmentNamingStrategy="NAME">
+  <email:imap-connection host="imap.gmail.com" port="993" user="user@gmail.com" password="mypassword">
+  </email:imap-connection>
+</email:imap-config>
+----
+
+The following example shows how to override the naming strategy at the *On New Email - IMAP* source level:
+
+. Select the *On New Email - IMAP* source in the Studio canvas.
+. On the source configuration screen, in the *Advanced* tab, for the *Attachment naming strategy* field select one of the following options:
+
+* *NAME*
+* *NAME_HEADERS*
+* *NAME_HEADERS_SUBJECT*
+
+The following screenshot shows the *Attachment naming strategy* override at the *On New Email - IMAP level* source:
+
+.Attachment naming strategy override source level
+image::email-attachment-2.png[Attachment naming strategy override source level]
+
+In the XML editor, the configuration `attachmentNamingStrategy="NAME"` is under the `listener-imap` tag:
+
+[source, xml, linenums]
+----
+<flow name="OnNewEmail-IMAP">
+		<email:listener-imap  config-ref="Email_IMAP" attachmentNamingStrategy="NAME">
+			<scheduling-strategy >
+				<fixed-frequency />
+			</scheduling-strategy>
+		</email:listener-imap>
+	</flow>
+----
+
+== Configure Enable Remote Search for IMAP Matcher Parameter
+
+When using *On New Email - IMAP* source and *Imap matcher* parameter, you can specify whether you want the filters to be applied server-side by enabling the *Enable remote search filter* parameter.
+
+This parameter is disabled by default because some email servers are not fully compliant with `rfc-3501` search terms. Enabling the parameter decreases traffic by reducing the amount of emails brought to client-side for processing.
```

Comment:
```suggestion
This parameter is disabled by default because some email servers are not fully compliant with `rfc-3501` search terms. Enabling the parameter decreases traffic by reducing the number of emails sent to the client-side for processing.
```